openml/datasets/functions.py

```diff
@@ -165,6 +165,30 @@ def _get_cache_directory(dataset: OpenMLDataset) -> str:
     return _create_cache_directory_for_id(DATASETS_CACHE_DIR_NAME, dataset.dataset_id)
 
 
+def list_qualities() -> list:
```

Comment:
Sorry for not catching this the first time, but please indicate it is a `List[str]` in the type hint. Currently it is hard to tell what is being returned exactly.